content/docs/04-clusters/03-edge/02-edge-configuration/35-installer-reference.md

```diff
@@ -155,6 +155,52 @@ Network settings specific to the network interface of the edge host. You can con
 | `networkInterface.gateway` | The network gatway IP address. |
 | `networkInterface.nameserver` | The IP address of the DNS nameserver this interface should route requests to.| 
 
+## User ID (UID) Parameters
+
+The User ID(UID) is generated by a specific priority sequence. The below table outlines the priority order from top to bottom, to generate a UID. The UID generation starts with priority 1, the device `name`, followed by attributes within the `deviceUID`, then obtaining the UID from the `/sys/class/dmi/id/product_uuid` file, and finally generating a random UUID if all other methods are unsuccessful.
+
+| Priority | Generate Method for UID                       | Description                                                                                           |
+|----------|------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| 1        | `name`                                    | The device name used as the primary identifier for the UID.                                        |
+| 2        | `deviceUID`                    | Other attributes within the `deviceUID`, such as product or board serial numbers. |
+| 3        | `/sys/class/dmi/id/product_uuid` | The UID obtained by reading the content of the `/sys/class/dmi/id/product_uuid`            |
+| 4        | `Random UUID`                                    | If all other methods fail, a random UUID is generated and used as the UID.                            |
+
+
+By default, the product UID path is set to `/sys/class/dmi/id/product_uuid`. To modify this path and use other attributes within the same folder, such as the product or board serial, use `regex` parameter. For example, instead of the default path `/sys/class/dmi/id/product_uuid`, you can use the board Serial Number path `/sys/class/dmi/id/board_serial` by applying a regex parameter. Refer to [syntax](https://github.com/google/re2/wiki/Syntax) for writing regex in the correct format.
```

Comment:
```suggestion
By default, the product UID path is set to `/sys/class/dmi/id/product_uuid`. To modify this path and use other attributes within the same folder, such as the product or board serial, use the `regex` parameter. For example, instead of the default path **/sys/class/dmi/id/product_uuid**, you can use the board Serial Number path **/sys/class/dmi/id/board_serial** by applying a regex parameter. Refer to the [regex syntax](https://github.com/google/re2/wiki/Syntax) reference guide to learn more.
```

File paths are in bold per our style guide.